README_zh_CN.md

```diff
@@ -15,8 +15,8 @@
 + 从文件导入设置
 
 # 缺点
-+ 仅测试过 W820NB双金标版 和 W200BT Plus  
-  (也许能用在 W820NB Plus 和 W820NB 上)  
++ 仅测试了 W820NB、W820NB双金标版 和 W200BT Plus
```

Comment:
Thank you for using google translate for it! However, I prefer to not make unnecessary changes here.
```suggestion
+ 仅测试过 W820NB、W820NB双金标版 和 W200BT Plus  
```